service/src/main/java/bio/terra/workspace/service/workspace/flight/RemoveUserFromWorkspaceFlight.java

```diff
@@ -0,0 +1,50 @@
+package bio.terra.workspace.service.workspace.flight;
+
+import bio.terra.stairway.Flight;
+import bio.terra.stairway.FlightMap;
+import bio.terra.workspace.common.utils.FlightBeanBag;
+import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
+import bio.terra.workspace.service.iam.model.WsmIamRole;
+import bio.terra.workspace.service.job.JobMapKeys;
+import java.util.UUID;
+
+public class RemoveUserFromWorkspaceFlight extends Flight {
+
+  public RemoveUserFromWorkspaceFlight(FlightMap inputParameters, Object applicationContext) {
+    super(inputParameters, applicationContext);
+
+    FlightBeanBag appContext = FlightBeanBag.getFromObject(applicationContext);
+
+    UUID workspaceId =
+        UUID.fromString(inputParameters.get(WorkspaceFlightMapKeys.WORKSPACE_ID, String.class));
+    AuthenticatedUserRequest userRequest =
+        inputParameters.get(JobMapKeys.AUTH_USER_INFO.getKeyName(), AuthenticatedUserRequest.class);
+    String userToRemove = inputParameters.get(WorkspaceFlightMapKeys.USER_TO_REMOVE, String.class);
+    WsmIamRole roleToRemove =
```

Comment:
Have I been sinning by serializing enums directly? It seems to work AFAICT.

Reply:
IIRC they would work normally, but there was an enum serialization issue when flights were recovered. I haven't kept up to date with Stairway happenings, I'll defer to @ddietterich if this has been fixed or is still an issue

Reply:
I think that serializing and deserializing standalone enums works like:
```
map.put("foo", MyEnum.FOO);
MyEnum anEnum = map.get("foo", MyEnum.class);
```
The issue I was aware of was using collections of enums. `List<MyEnum>` would not deserialize properly with `List.class`. However, now that we can use `TypeReference`, you can get that to work also.

There is a case that is untested in the Stairway regression tests: an enum embedded in another class. However, those are always tagged with the type and I believe we use them frequently.

Reply:
sgtm, switched to directly serializing this enum